tools/OldOutputTypeIndex.ps1

```diff
@@ -0,0 +1,58 @@
+param(
+    [Parameter(Mandatory = $false)]
+    [string] $BuildConfig = "Release",
+    [Parameter(Mandatory = $false)]
+    [string] $OutputFile = "outputtypes.json"
+)
+Install-Module Az.Accounts -Repository PSGallery -Force -Scope CurrentUser
+$AzPreviewPath = Get-Item $PSScriptRoot\AzPreview\AzPreview.psd1
+Import-LocalizedData -BindingVariable ModuleMetadata -BaseDirectory $AzPreviewPath.DirectoryName -FileName $AzPreviewPath.Name
+$ModulePath = ($env:PSModulePath -split ';')[0]
+$outputTypes = New-Object System.Collections.Generic.HashSet[string]
+$jsonData = Get-Content $OutputFile | ConvertFrom-Json
```

Comment:
Why need to rely on the existing outputtypes.json, can we build it in just one script?